src/utils/test-utils.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, RenderOptions } from "@testing-library/react";
+import React, { FC, ReactElement } from "react";
+import { I18nextProvider } from "react-i18next";
+import i18n from "../i18n/test_config";
+
+const AllTheProviders: FC<{ children: React.ReactNode }> = ({ children }) => {
+  return <I18nextProvider i18n={i18n}>{children}</I18nextProvider>;
+};
+
+const customRender = (
+  ui: ReactElement,
+  options?: Omit<RenderOptions, "wrapper">
+) => render(ui, { wrapper: AllTheProviders, ...options });
+
+export * from "@testing-library/react";
```

Comment:
This I'm not sure if I like this line.
I'm a bit worried you might need more things and they will all end up in here and it's hard to see what's coming from where in the tests. I.e. https://mswjs.io, https://testing-library.com/docs/user-event/setup/, etc.

Everything else looks like a great improvement.

Reply:
Was done as described here:
https://testing-library.com/docs/react-testing-library/setup/

> I'm a bit worried you might need more things and they will all end up in here

msw setup is in `testServer.ts`, so it's already split up.

Not sure if I understand correctly, but the main reason is the render & it saves using other imports / using the "wrong" `render` by just using everyhing from `test-utils`.